spec/requests/rate_limiting_spec.rb

```diff
@@ -34,6 +34,51 @@
     end
   end
 
-  xit 'POST /otp-sign-in'
-  xit 'POST /otp-sign-in/verify'
+  context "rate limit /api/ endpoints by auth token" do
+    it "throttles over 1000 requests within 5 minutes" do
+      freeze_time do
+        1000.times do
```

Comment:
I think when the statements endpoint actually does something that hits the DB these test will be pretty slow. In NPQ reg we mock out the limits to get around that, is it worth doing something similar here?

Reply:
Updated the tests to what we have for NPQ